components/scripts/lib/build_scan.sh

```diff
@@ -166,6 +166,8 @@ fetch_and_read_build_scan_data() {
     args+=(-m "${mapping_file}")
   fi
 
+  args+=("--proxy-settings-file" "${SCRIPT_DIR}/proxy.settings")
```

Comment:
Minor: I think we should use `.properties` here so that editors/IDEs can help with editing.